derivatives/labels/sub-test001/anat/sub-test001_T1w_lesion-manual.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
where is this data coming from? is it a public dataset?  if not, why not simply using a subject from spine-generic?

Reply:
It's just the copy of the testing data that was already in ivadomed. I do need to add the new data that will be used in the new tests. There will still be a single subject. I don't think it is a public dataset (this  [PR](https://github.com/ivadomed/ivadomed/pull/103) ).

Reply:
>  I don't think it is a public dataset

can we replace it with another subject then

Reply:
also: if you don't rebase the unused binaries, please make sure to "squash and merge" for merging this PR

Reply:
I'm looking into it with the test

Reply:
I think we can. Just need some refactoring (because previous test were build to use sub-test001_lesion-manual.nii.gz and we don't have the lesion manual suffix anymore)

Reply:
> we don't have the lesion manual suffix anymore

can't you just create a dummy seg?

Reply:
Maybe. I am trying with the available csf_seg given in the ivadomed/data_example_spinegeneric repo

Reply:
ok, i'm getting confused now, i thought you were looking for a file with suffix `_lesion-manual`?

i would suggest you list, in the PR's body, the files that are needed by your tests. Then, we can all think and find the best solution for the listed files (in terms of size, availability, need for manual intervention, etc.)

Reply:
Oh sorry I will list them. 

I was using the csf segmentation as a dummy seg. (I renamed it T2w_lesion_manual).

Reply:
> I renamed it T2w_lesion_manual

that’s exactly the kind of stuff i’d like to avoid. it might bring confusion to a future student

Reply:
@lrouhier: For the previous testing data: what I did was to create a "fake lesion", ie a blob, in the spinal cord (the subject has healthy).